examples/tests/test_offchain_error_cases.py

```diff
@@ -295,6 +295,8 @@ def test_travel_rule_limit_validation(sender_app, receiver_app):
     request = minimum_required_fields_request_sample(sender_app, receiver_app, amount=10)
 
     resp = send_request(request, sender_app, receiver_app, "failure")
+    assert resp.cid
```

Comment:
nit: redundant

Reply:
was intent to add this one first, to ensure it is not None, then compare it with request cid, so that it is clear to reader, they are not None